CONTRIBUTING.md

```diff
@@ -1,30 +1,49 @@
 # Contributing
 
-We highly welcome community contributions and encourage contributions.
+## Our philosophy
+
+We aim for this book to be a major resource for introducing people to the field of single-cell sequencing data analysis and ensuring that newcomers and experienced analysis do things right.
+To ensure our recommendations reflect best practices, we try to rely exclusively on external and independent benchmarks.
+
+## How can I contribute?
+
+We warmly welcome contributions, especially those that help us keep our best practices up to date!
+However, as mentioned [above](#our-philosophy), these recommendations are held to high standards.
+
+If you're proposing new tutorials, corrections, or extensions, we strongly recommend opening an issue first to discuss your ideas with us.
+We're always eager to learn about the latest developments and will likely have follow-up questions — but we're genuinely excited to collaborate.
+
+We also encourage contributions in the form of [quizzes and flashcards](#create-custom-quizzes-and-flashcards).
+Our collection of self-assessment questions is always growing, and community support in expanding them is greatly appreciated.
+The same applies to these contributions: Just open an issue and let's get in touch!
+
+[![Open an Issue](https://img.shields.io/badge/Open%20Issue-blue?logo=github)](https://github.com/theislab/single-cell-best-practices/issues/new?title=Your+Issue+Title&body=Describe+your+issue+here)
 
 ## Book architecture
 
-The following depicts the rough structure of the book inside the `jupyter_book` folder.
-Beyond several configuration files, all chapters are stored inside corresponding folders, such as for example, `conditions`.
-Inside, the corresponding notebooks, together with the associated `reference.bib` and `environment.yml` files, are stored.
+The structure of the book is organized within the `jupyter_book` folder.
```

Comment:
Weird sentence. I'd suggest that we rephrase this to the source of the book being in that folder or so